aws/resource_aws_athena_named_query_test.go

```diff
@@ -98,16 +98,26 @@ resource "aws_s3_bucket" "hoge" {
   force_destroy = true
 }
 
+resource "aws_kms_key" "test" {
+  deletion_window_in_days = 7
+  description             = "Terraform Acceptance Testing"
+}
+
+resource "aws_athena_workgroup" "test" {
+  name = "tf-athena-workgroup-%s-%d"
+}
+
 resource "aws_athena_database" "hoge" {
   name   = "%s"
   bucket = "${aws_s3_bucket.hoge.bucket}"
 }
 
 resource "aws_athena_named_query" "foo" {
   name        = "tf-athena-named-query-%s"
+  workgroup   = "${aws_athena_workgroup.test.id}"
```

Comment:
Instead of modifying an existing acceptance test and its configuration can you please split this into its own test and test configuration? This will verify that this resource code update does not cause any regressions with existing Terraform configurations. Copy-pasting the existing test and configuration is totally fine. See also the [`Enhancement/Bug Fix to a Resource` section of the Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#enhancementbugfix-to-a-resource). Thanks!